docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -155,9 +144,8 @@ spec:
       enabled: true
 ----
 
-. If the source environment does not enable TLS Everywhere, modify
-spec:tls section with the following override before applying it:
-
+. If your {OpenStackShort} environment does not enable TLS-e, in the
```

Comment:
This step says that you only have to set the enabled parameter to "false" in the spec:tls section, but there's also a spec for the osp-secret that has all the services' enabled parameters set to false. 
Should we have another step below this one that says, "If your RHOSP environment does not enable TLS-e, in the osp-secret spec, set the enabled parameter in each service to "false"?

Reply:
Also, are you modifying the spec:tls section in the openstack_control_plane.yaml file in steps 6 and 7?

Reply:
>  there's also a spec for the osp-secret that has all the services' enabled parameters set to false

I'm not aware of this. Can you link what you're referring to?

> are you modifying the spec:tls section in the openstack_control_plane.yaml file in steps 6 and 7?

This is about modifying the content of the OpenStackControlPlane CR creation snippet below, the one starting `oc apply`. We don't have the openstack_control_plane.yaml file in the Adoption guide presently but the `oc apply` snippet has the same purpose.

Reply:
> > there's also a spec for the osp-secret that has all the services' enabled parameters set to false
> 
> I'm not aware of this. Can you link what you're referring to?

In step 7, there are 2 code blocks one after another.
https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/adopting_a_red_hat_openstack_platform_17.1_deployment/migrating-databases-to-the-control-plane_storage-requirements#deploying-backend-services_migrating-databases

Reply:
> > there's also a spec for the osp-secret that has all the services' enabled parameters set to false
> 
> I'm not aware of this. Can you link what you're referring to?
> 
I realize now that the "oc apply -f - <<EOF" command does not belong in step 7. It needs to be shifted up to step 5.

Reply:
@klgill Step 6 should come before the `oc apply` though ("modify tls:spec section *before applying it*"). So rather than moving the `oc apply` to step 5, let's get rid of step 5 by moving that description to step 7. (Then step 6 becomes step 5 and step 7 becomes step 6 :).)

Reply:
Also Step 7 now combines the override for nonTLS with the actual `oc apply`. I think it would make more sense for what is now Step 6 to be the "tls:spec" section overrides, both for TLS and non-TLS variants, and then after that comes the deployment step with `oc apply`.